packages/components/Button/src/FAB/FABTokens.ts

```diff
@@ -75,4 +75,5 @@ export const defaultFABTokens: TokenSettings<FABTokens, Theme> = (t: Theme) =>
         spacingIconContentBefore: globalTokens.size80,
       },
     },
+    shadowToken: t.shadows.shadow8,
```

Comment:
I don't think this is a necessary for either iOS or android shadows - iOS will be referencing the FABTokens.ios.ts file instead, and the shadowToken shouldn't have any effect on Android shadows

Reply:
Agreed, removing it